modules/core/src/main/java/org/apache/ignite/internal/processors/tracing/Span.java

```diff
@@ -26,64 +26,60 @@
  * status, time events, attributes, links.
  * You can use tracing to debug errors and latency issues in your applications.
  */
-public interface Span {
+public interface Span extends SpiSpecificSpan {
```

Comment:
Do we need inheritance here?
- We already delegate all calls to spiSpecificSpan 
- Looks like it would be more clear to distinguish that regular Span is not SPI specific (Ignite logic will work independent of SPI choice)

Reply:
Fixed.